go/vt/vttablet/onlineddl/executor.go

```diff
@@ -765,8 +766,93 @@ func (e *Executor) terminateVReplMigration(ctx context.Context, uuid string) err
 	return nil
 }
 
+func (e *Executor) killQueriesOnTable(ctx context.Context, tableName string) error {
```

Comment:
Since this kills both queries and transactions/connections, it should have a comment describing it and a better name would be something like `killAccessorsOfTable` or `killTableLockHolders`. IMO the former is better because we will also want to kill anything that's trying to get a lock which has not yet been granted (we'll need to clear out the lock holders along with the request queue).

Reply:
Renamed as `killTableLockHoldersAndAccessors`, and added a lot of comments.